doc/getting_started/index.rst

```diff
@@ -3,319 +3,409 @@
+      :ref:`installing 3rd-party toolchains<gs_toolchain>`.
 
-   .. code-block:: console
+   .. group-tab:: Windows
 
-      # Linux and macOS
-      source zephyr-env.sh
+      The Zephyr SDK is not supported on Windows.  See instructions for
+      :ref:`installing 3rd-party toolchains<gs_toolchain>`.
+
+.. _getting_started_run_sample:
+
+.. rst-class:: numbered-step
+
+Build the Blinky Application
+****************************
 
-      # Windows
-      zephyr-env.cmd
+The sample :ref:`blinky-sample` blinks an LED on the target board.  By
+building and running it, we can verify that the environment and tools
+are properly set up for Zephyr development.
 
-#. Build the Hello World sample for the ``reel_board``:
+#. Set build environment variables:
 
-   .. zephyr-app-commands::
-      :app: samples/hello_world
-      :board: reel_board
-      :goals: build
+   .. tabs::
 
-The main build products will be in :file:`build/zephyr`;
-:file:`build/zephyr/zephyr.elf` is the Hello World application binary in ELF
-format. Other binary formats, disassembly, and map files may be present
-depending on your board.
+      .. group-tab:: Ubuntu
 
-The other sample applications in :zephyr_file:`samples` are documented in
-:ref:`samples-and-demos`. If you want to re-use an existing build directory for
-another board or application, you need to pass ``-p=auto`` to ``west build``.
+         .. code-block:: bash
 
-Run the Application by Flashing to a Board
-==========================================
+            cd ~/zephyrproject/zephyr
+            source zephyr-env.sh
 
-Most "real hardware" boards supported by Zephyr can be flashed by running
-``west flash``. However, this may require board-specific tool installation and
-configuration to work properly.
+      .. group-tab:: macOS
 
-See :ref:`application_run` in the Application Development Primer and your
-board's documentation in :ref:`boards` for additional details.
+         .. code-block:: bash
 
-Run the Application in QEMU
-===========================
+            cd ~/zephyrproject/zephyr
+            source zephyr-env.sh
 
-On Linux and macOS, you can run Zephyr applications in emulation on your host
-system using QEMU when targeting either the x86 or ARM Cortex-M3 architectures.
+      .. group-tab:: Windows
 
-To build and run Hello World using the x86 emulation board configuration
-(``qemu_x86``), type:
+         .. code-block:: bash
 
-.. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
-   :host-os: unix
-   :board: qemu_x86
-   :goals: build run
+            cd %HOMEPATH%/zephyrproject/zephyr
+            zephyr-env.cmd
 
-To exit, type :kbd:`Ctrl-a`, then :kbd:`x`.
+#. Build the blinky sample. Specify **your board name** in the command below:
 
-Use ``qemu_cortex_m3`` to target an emulated Arm Cortex-M3 instead.
+   .. code-block:: bash
 
-.. _gs_posix:
+      west build -p -b <your-board-name> samples/basic/blinky
```

Comment:
Mixed feelings about using --pristine here, as it instills a "bad" habit of never doing incremental builds.

But not asking for a change, just pointing this out.

Reply:
Yes, I know that.  But for this simplified instruction path it was an easy way to eliminate problems seen by test subjects that chose the wrong board and then tried specifying the right board without cleaning out the build folder (bad things happen if you change the board or application).  I could change this to ``-p auto`` I suppose...

Reply:
Right, I'm not asking for a change, but maybe this means 'doing incremental builds' ought to be a prominent 'beyond the GSG' topic.